db/schema.rb

```diff
@@ -182,6 +186,7 @@
     t.integer "division_id"
     t.boolean "eligible", default: false
     t.integer "slots_available", default: 0
+    t.boolean "look_for_member", default: true
```

Comment:
It would be preferred if this row was called `looking_for_members` for consistency.